--- a/chrome/app/theme/chromium/BRANDING
+++ b/chrome/app/theme/chromium/BRANDING
@@ -1,10 +1,10 @@
-COMPANY_FULLNAME=The Chromium Authors
-COMPANY_SHORTNAME=The Chromium Authors
-PRODUCT_FULLNAME=Chromium
-PRODUCT_SHORTNAME=Chromium
-PRODUCT_INSTALLER_FULLNAME=Chromium Installer
-PRODUCT_INSTALLER_SHORTNAME=Chromium Installer
-COPYRIGHT=Copyright @LASTCHANGE_YEAR@ The Chromium Authors. All rights reserved.
-MAC_BUNDLE_ID=org.chromium.Chromium
+COMPANY_FULLNAME=The Arrow Authors
+COMPANY_SHORTNAME=The Arrow Authors
+PRODUCT_FULLNAME=Arrow
+PRODUCT_SHORTNAME=Arrow
+PRODUCT_INSTALLER_FULLNAME=Arrow Installer
+PRODUCT_INSTALLER_SHORTNAME=Arrow Installer
+COPYRIGHT=Copyright @LASTCHANGE_YEAR@ The Arrow Authors. All rights reserved.
+MAC_BUNDLE_ID=org.arrow.browser
 MAC_CREATOR_CODE=Cr24
 MAC_TEAM_ID=
